frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -103,9 +101,23 @@ const footerVariantStyles = {
   `,
 };
 
-export const ParametersWidgetContainer = styled(FullWidthContainer)`
-  padding-top: ${space(1)};
-  padding-bottom: ${space(1)};
+export const ParametersWidgetContainer = styled(FullWidthContainer)<{
+  hasVisibleParameters: boolean;
+}>`
+  position: sticky;
+  top: 0;
+  left: 0;
+  width: 100%;
+  z-index: 3;
+
+  background-color: ${color("white")};
+
+  ${props =>
+    props.hasVisibleParameters &&
+    css`
+      padding-top: ${space(1)};
+      padding-bottom: ${space(1)};
+    `}
```

Comment:
@WiNloSt this should address [your comment](https://github.com/metabase/metabase/pull/38634#discussion_r1487418364) on of the previous PRs